CONTRIBUTING.md

```diff
@@ -4,5 +4,6 @@ If you have discovered a bug or have a feature suggestion, feel free to create a
 
 If you'd like to make some changes yourself, see the following:
 1. [Fork](https://help.github.com/articles/fork-a-repo/) this repository to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your local device
-2. Build and start the application: `yarn dev`
-3. Finally, submit a [pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) with your changes!
+2. Run `npm install` to download required packages.
```

Comment:
I agree! Can you actually change this to include a step about needing `yarn` (npm i -g yarn) and then change `npm install` to `yarn`.